docs/source/app_developers_guide/docker.rst

```diff
@@ -2,696 +2,672 @@
+      sawtooth-validator-default | [2018-03-08 22:55:10.562 DEBUG    genesis] Deleting genesis data.
+      sawtooth-validator-default | [2018-03-08 22:55:10.564 DEBUG    selector_events] Using selector: ZMQSelector
+      sawtooth-validator-default | [2018-03-08 22:55:10.565 INFO     interconnect] Listening on tcp://eth0:8800
+      sawtooth-validator-default | [2018-03-08 22:55:10.566 DEBUG    dispatch] Added send_message function for connection ServerThread
+      sawtooth-validator-default | [2018-03-08 22:55:10.566 DEBUG    dispatch] Added send_last_message function for connection ServerThread
+      sawtooth-validator-default | [2018-03-08 22:55:10.568 INFO     chain] Chain controller initialized with chain head: 60e79c91757c73185b36802661833f586f4dd5ef3c4cb889f37c287921af8ad01a8b95e9d81af698e6c3f3eb7b65bfd6f6b834ffc9bc36317d8a1ae7ecc45668 (block_num:0, state:53d38378e8c61f42112c39f9c84d42d339320515ef44f50d6b4dd52f3f1b9054, previous_block_id:0000000000000000)
+      sawtooth-validator-default | [2018-03-08 22:55:10.569 INFO     publisher] Now building on top of block: 60e79c91757c73185b36802661833f586f4dd5ef3c4cb889f37c287921af8ad01a8b95e9d81af698e6c3f3eb7b65bfd6f6b834ffc9bc36317d8a1ae7ecc45668 (block_num:0, state:53d38378e8c61f42112c39f9c84d42d339320515ef44f50d6b4dd52f3f1b9054, previous_block_id:0000000000000000)
+      ...
 
-After shutdown has completed, run this command:
+This terminal window will continue to display log messages as you run commands
+in other containers.
 
-.. code-block:: console
+.. note::
 
-  % docker-compose -f sawtooth-default.yaml down
+   If you need to reset the environment environment for any reason, see
+   "Stopping Sawtooth".
 
 
-Logging Into The Client Container
-=================================
+Step 4: Log Into the Client Container
+=====================================
 
-The client container is used to run Sawtooth commands, which is the usual
-way to interact with validators or validator networks.
+Sawtooth includes commands that act as a client application. The client
+container is used to run these Sawtooth commands, which interact with the
+validator through the REST API.
 
-Log into the client container by opening a new terminal window and
-running the following command. Note that ``sawtooth-shell-default`` specifies
-the client container name.
+To log into the client container, open a new terminal window and run the
+following command:
 
 .. code-block:: console
 
-  % docker exec -it sawtooth-shell-default bash
+  $ docker exec -it sawtooth-shell-default bash
+  root@75b380886502:/#
 
+In the following steps, the simplified prompt ``#`` is used for the commands
+that run from the terminal window for the client container. The ``$`` prompt
+denotes commands that should be executed from a terminal window on your host
+system. The full prompt ``root@21bb5af673a8:/#`` shows commands that run in the
+validator container.
```

Comment:
I am not a huge fan of this convention, as it is pretty arbitrary. Maybe ``root@client #`` for client, ``host $``, and ``root@validator #`` would be more meaningful?

Reply:
Good idea! Will do.